h2o-r/h2o-package/R/export.R

```diff
@@ -337,6 +337,7 @@ h2o.saveModelDetails <- function(object, path="", force=FALSE) {
 #'
 #' @param grid_directory A character string containing the path to the folder for the grid to be saved to.
 #' @param grid_id A chracter string with identification of the grid to be saved.
+#' @param save_frames Save data frames used by this grid as well
```

Comment:
Usually in R API, first the type expected is described. Not always, but most of the times yes. Just noticed it.